netci.groovy

```diff
@@ -762,7 +762,7 @@ def static setMachineAffinity(def job, def os, def architecture, def options = n
                 label('Windows.10.Amd64.ClientRS4.DevEx.Open')
             }
         } else {
-            Utilities.setMachineAffinity(job, os, 'arm64-windows_nt')
+            Utilities.setMachineAffinity(job, 'windows.10.arm64.open')
```

Comment:
I don't think this will work. I think you need:
```
job.with {
    label('Windows.10.Arm64.Open')
}
```

Is there a "correct" casing (upper/lower)?

Reply:
- The queue API is case insensitive, but I think we're trending towards all lower case.  Use whatever you like here, case insensitivity (since Service Bus Messaging queues are) is here to stay until we change our queuing technology.
- I believe the job.with format though is assuming that the build will occur on the ARM machine, not just a test run as this was being used before.  I think the setMachineAffinity is still the right way to go, but I defer to @mmitche .

Reply:
Casing shouldn't matter.

Reply:
The job.with format is identical to the setMachineAffinity format underneath.